src/components/graph/util/center-graph-button.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import CenterFocusIcon from '@material-ui/icons/CenterFocusStrong';
+import React, { useCallback } from 'react';
+import { ControlButton, useZoomPanHelper } from 'react-flow-renderer';
+
+const CenterGraphButton = ({ selectedNode }) => {
+    const { setCenter } = useZoomPanHelper();
+
+    const focusNode = useCallback(() => {
+        // if no selected node, center on Root
+        const x = selectedNode ? selectedNode.position.x : 0;
+        const y = selectedNode ? selectedNode.position.y : 0;
+        const zoom = 1;
```

Comment:
Do we really want to overwrite the zoom value when centering on node ? that's a good question :thinking:

Reply:
I agree modifying zoom might not be a good idea. I modified the code to keep the current zoom.

Reply:
Nice and simple fix GG!